### ISSUE TITLE
This is awesome!

### ISSUE THREAD

**Issue by ShaggyDude** [thomaspark/flexboxfroggy#13]
I didn't know where to add a comment. 
This is awesome. Thanks!

**Comment by cbas** [comment]
:+1: Just did the exercise with 10+ students ... thanks for creating this @thomaspark :heart:

**Comment by malchata** [comment]
I've been giving talks on Flexbox, and I'm definitely adding this as a slide if I do another speaking date. You rule!

**Comment by mfdj** [comment]
:sparkles: :sparkling_heart: :sparkles:

**Comment by NouranMahmoud** [comment]
I loved it, really awesome! :+1:

**Comment by jameskopacz** [comment]
:+1:

**Comment by kingstenbanh** [comment]
Thank you for creating such a delightful and enjoyable app to learn Flexbox. You rock!

**Comment by arbatskiy** [comment]
:+1: yep, the app is fantastic, thank you for creating this!

**Comment by sushant23** [comment]
This is so awesome really cool.... :+1:

**Comment by theJollySin** [comment]
If only *all* learning was this much fun!  Great work!

**Comment by based2** [comment]
+1

**Comment by matiasgq** [comment]
Excellent job brother, very creative 👏👏👏 @thomaspark

**Comment by webleandroromano** [comment]
Awesome!

**Comment by Zzing-gu** [comment]
The best way of teaching Flex!

**Comment by crumpkl** [comment]
Amazing! A great visual way to learn Flex! Thanks!

**Comment by jam1015** [comment]
yes! probably the best tutorial I've encountered for any programming topic

**Comment by Nevin06** [comment]
Nice game.

**Comment by arshit09** [comment]
Great way to learn. Easy and simple way to practice.
